batch/batch/driver/instance.py

```diff
@@ -159,7 +159,7 @@ async def activate(self, ip_address, timestamp):
         assert self._state == 'pending'
 
         rv = await check_call_procedure(
-            self.db, 'CALL activate_instance(%s, %s, %s);', (self.name, ip_address, timestamp)
+            self.db, 'CALL activate_instance(%s, %s, %s);', (self.name, ip_address, timestamp), 'activate_instance'
```

Comment:
I don't believe this works with this function signature. The `check_call_procedure` also seems a bit random where it is currently defined. Let's turn that function into a method on the `Database` class, make that method signature work with the query timing, and delete the file it's currently defined in.